Gemfile

```diff
@@ -7,7 +7,8 @@ gem 'activesupport', '~> 7.0'
 gem 'assembly-image', '~> 2.0' # ruby-vips is used by 2.0.0 for improved image processing
 gem 'assembly-objectfile', '~> 2.1'
 gem 'config', '~> 2.2'
-gem 'dor-services-client', '~> 13.0'
+# gem 'dor-services-client', '~> 13.0'
+gem 'dor-services-client', github: 'sul-dlss/dor-services-client', branch: 'reset'
```

Comment:
this clearly isn't meant to be merged yet.

Reply:
if this needs to be here, even temporarily, maybe a comment explaining that we can't cut gem release until xxx happens?

Reply:
Sorry @ndushay. This PR should have been a draft.